HISTORY.rst

```diff
@@ -108,6 +108,10 @@ Unreleased Changes (3.9)
       'watersheds' to match the name of the vector file (including the suffix).
     * Added pour point detection option as an alternative to providing an
       outlet features vector.
+* Forest Carbon Edge Effects
+    * Fixed a broken link to the local User's Guide
+* Finfish
+    * Fixed a bug where the suffix input was not being used for output paths.
```

Comment:
Should Finfish go before Forest Carbon!?

Reply:
doh!